auth/tests/integration/smoke_test.py

```diff
@@ -43,3 +45,16 @@
     assert token.access_token is not None
     assert token.access_token_duration != 0
     assert token.access_token_acquired_at != datetime.datetime(1970, 1, 1)
+
+
+@pytest.mark.asyncio  # type: ignore
+async def test_sign_blob(creds: str) -> None:
+    data = 'Testing Can be confidential!'
+    payload = encode(data)
+
+    async with aiohttp.ClientSession(conn_timeout=10, read_timeout=10) as s:
+
+        iam_credentials = IamCredentialsClient(service_file=creds, session=s)
+        signed_data = iam_credentials.sign_blob(payload)
```

Comment:
This should be `signed_data = await iam_credentials...`

Reply:
done